android/java/org/chromium/chrome/browser/ntp/BraveNewTabPageLayout.java

```diff
@@ -588,24 +597,52 @@ protected void onAttachedToWindow() {
 
         if (ChromeFeatureList.isEnabled(BraveFeatureList.BRAVE_NEWS)) {
             if (mSettingsBar != null) {
-                mSettingsBar.setVisibility(View.VISIBLE);
                 mSettingsBar.setAlpha(0f);
+                mSettingsBar.setVisibility(View.INVISIBLE);
             }
             InitBraveNewsController();
+            initNews();
+            Tab tab1 = BraveActivity.getBraveActivity().getActivityTab();
```

Comment:
A check `if (BraveActivity.getBraveActivity() != null` should probably be before that line

Reply:
@SergeyZhukovsky the whole "tab1" block was useless there i think it was part of some dev logs . Same as below